scripts/hab_conf_transfer_to_Q.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Transfer `habitat_confirmations.xls` data to `form_fiss_2023.gpkg`"
```

Comment:
title makes sense but be sure to recognize that this type of workflow is flexible to move data from any file to any file.  We could pull data from a quick little csv and put it in the Q file or we could grab data from the database, write it to the gpkg then export it to add to the submission templates.  often it is just a matter of which sources we start with and which we finish with and not that different from one to the next.  We won't need a rmd for each one but thanks for making this as it is helpful for us both to learn